pkg/bench/bpfstats.go

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package bench
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+)
+
+func EnableBpfStats() {
+	err := os.WriteFile("/proc/sys/kernel/bpf_stats_enabled", []byte("1"), 0666)
```

Comment:
Is the perm mode ok? maybe restrict it a bit

Reply:
that permission will be used only if the file does not exist and needs to be created, so we are fine here